_docs/_latest/use/scanner.md

```diff
@@ -16,7 +16,7 @@ Forseti Scanner works on a data model, so before you start using Scanner, you'll
 Instructions on how to [create a model]({% link _docs/latest/use/model.md %}).
 
 To configure which scanners to run, see 
-[Configuring Forseti: Configuring Scanner]({% link _docs/latest/configure/configuring-forseti.md %}#configuring-scanner).
+[Configuring Forseti: Configuring Scanner]({% link _docs/latest/configure/scanner/index.md %}).
```

Comment:
Not here, but please (everywhere) insert a `---` after the page title content and before the first heading for consistency reasons (see /concepts).

Reply:
Updated all the files under configure to use `---`, will leave the rest out for other reviewers to avoid merge conflicts.